<launch>
    <node pkg="orb_slam2" type="feature_detector" name="feature_detector" output="screen">
      <param name="publish_image_" value="true" />
      <remap from="image" to="/image_rect"/>
      <remap from="camera_info" to="/cam/camera_info"/>
    </node> 

    <node pkg="orb_slam2" type="ros_mono" name="slam" output="screen" args="/home/biorob/catkin_ws/src/orb_slam2/Vocabulary/ORBvoc.txt /home/biorob/catkin_ws/src/orb_slam2/Examples/Monocular/test.yaml">
      <remap from="image" to="/image_rect"/>
      <remap from="features" to="/features"/>
    </node>
    
     <node pkg="rosbag" type="play" name="player" output="screen" args="--clock --loop /home/biorob/catkin_ws/src/orb_slam2/runs/succesfullInit_2016-03-03-13-10-14.bag"/>
    
  
<!--
    <node pkg="rosbag" type="record" name="recorder" output="screen" args="/image_rect -o succesfullInit"/>
  -->
</launch>
